### DISCUSSION TITLE
running getAnnotationCorrelation(ChromVar) within a seurat object

### DISCUSSION THREAD

**Discussion by debbie28** [stuart-lab/signac#525] [Q&A] [answered]
Hello,

I used Runchromvar to obtain variability for all motifs/across cells in my scATAC data. Now I want to identify groups of motif whose variability is correlated across cells. Ideally, I want to use the getAnnotationCorrelation function in ChromVar to obtain pearson correlation values between different motifs. However, I am unsure about how to run this function within the current signac object. I feel this is something that I am missing. Thanks for your help.

**Comment by timoast** [comment, marked as answer]
We only support computing the chromVAR deviations through the Signac package, if you want to use other functionality from chromVAR you'll need to use their package instead: https://greenleaflab.github.io/chromVAR/index.html
